spec/features/user_views_book_page_spec.rb

```diff
@@ -25,4 +25,45 @@
     expect(page).to have_content('June 19, 2015')
     expect(page).to have_css("img[src='http://example.com/cool_cover.jpg']")
   end
+
+  describe "didn't sign in" do
+    it "can't leave a review for a book" do
+      book = create :book
+      visit book_path(book)
+      expect(page).to have_content('Please sign in to leave a review')
+      expect(page).to_not have_content('Leave a review')
+    end
+  end
+
+  describe 'signed in' do
+    it "sees other users reviews" do
+      book = create :book
+      user = create :user, email: 'foo@bar.com'
+      other_user = create :user, email: 'foo1@bar.com'
+      review = create :review, user_id: other_user.id, book_id: book.id
+      sign_in(user)
+      visit book_path(book)
+      expect(page).to have_content(review.comment)
+      expect(page).to have_content(review.user.email)
+    end
+
+    it "didn't write a review yet" do
+      book = create :book
+      user = create :user
+      sign_in(user)
+      visit book_path(book)
+      expect(page).to have_content('Leave a review')
+    end
+
+    it "already write a review" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.